great_expectations/expectations/core/expect_column_distinct_values_to_equal_set.py

```diff
@@ -38,6 +38,10 @@ class ExpectColumnDistinctValuesToEqualSet(ColumnAggregateExpectation):
     expect_column_distinct_values_to_equal_set is a \
     [Column Aggregate Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations).
 
+    Column Aggregate Expectations are one of the most common types of Expectation.
```

Comment:
See previous edit recommendations and apply them to all topics containing similar text.